e2e-tests/testscript_test.go

```diff
@@ -206,6 +206,8 @@ func doRun(t *testing.T, experiments []string) {
 						}
 					}
 				}
+
+				time.Sleep(1 * time.Second)
```

Comment:
If this is needed this test is probably quite flakey already

Reply:
Yeah it is. This only happened when I brought in the change from your GRPC branch so I’m not sure why it made it flakey; I assume because of the background go routine now needing to start and get to a point of seving